### ISSUE TITLE
compile.error 1.1.0.3

### ISSUE THREAD

**Issue by andreitudor** [bytepaydev/Bytepay#4]
when i compile i get this error

                                    ^
masternodeman.cpp:922:17: error: ‘state’ was not declared in this scope
             if (state.IsInvalid(nDoS))
                 ^
masternodeman.cpp:922:17: note: suggested alternative:
In file included from /usr/include/boost/thread/future.hpp:26:0,
                 from /usr/include/boost/thread.hpp:24,
                 from util.h:24,
                 from script.h:19,
                 from core.h:10,
                 from net.h:9,
                 from masternodeman.h:11,
                 from masternodeman.cpp:1:
/usr/include/boost/thread/futures/future_status.hpp:26:12: note:   ‘boost::future_state::state’
       enum state { uninitialized, waiting, ready, moved, deferred };
            ^
In file included from script.h:19:0,
                 from core.h:10,
                 from net.h:9,
                 from masternodeman.h:11,
                 from masternodeman.cpp:1:
masternodeman.cpp:924:91: error: ‘tx’ was not declared in this scope
        LogPrintf("dsee - %s from %s %s was not accepted into the memory pool\n", tx.GetHash(
                                                                                  ^
util.h:158:39: note: in definition of macro ‘LogPrintf’
 #define LogPrintf(...) LogPrint(NULL, __VA_ARGS__)
                                       ^
masternodeman.cpp: At global scope:
masternodeman.cpp:932:5: error: expected unqualified-id before ‘else’
     else if (strCommand == "dseep") { //DarkSend Election Entry Ping
     ^
makefile.unix:201: recipe for target 'obj/masternodeman.o' failed
make: *** [obj/masternodeman.o] Error 1``

**Comment by tdiesler** [comment]
The above may fix the compile issue, but may also cause the daemon to crash like this

```
2018-05-24 22:52:15 ProcessBlock: ORPHAN BLOCK 339, prev=b7581c89d8938a1b5db635abccbf57383d5bdcd474742a7a495b2e1de814814e
2018-05-24 22:52:15 ProcessBlock: ORPHAN BLOCK 340, prev=c08e4555cc3e9cd679894ac908d7337162323d1a4862e841ae59983c332ca82e
2018-05-24 22:52:15 ProcessBlock: ORPHAN BLOCK 341, prev=0fbea5424d7f14bf01ad0d2f498f28a6c9996dd28bb060badb1d4def72f20b2b
2018-05-24 22:52:15 dseep - Asking source node for missing entry CTxIn(COutPoint(41d04143a7, 1), scriptSig=)
```

**Comment by tdiesler** [comment]
Yes, the daemon reliably keeps crashing after a while with that last message in the log. Tag v1.1.0.3 is not usable IMHO - not sure what the other master nodes run.

**Comment by tdiesler** [comment]
For completion, I documented what I did in this [gist](https://gist.github.com/tdiesler/2ee33f75f760f31a402189d7727e56dd)

**Comment by listenlight** [comment]
Sorry but I'm struggling to find the command to use to compile. The normal `./configure && make && make install` isnt available. Plus the gist you link to has a 404 now. Thanks.
